airgun/views/location.py

```diff
@@ -43,3 +49,95 @@ def is_displayed(self):
                 and self.breadcrumb.locations[0] == 'Locations'
                 and self.breadcrumb.read() == 'New Location'
         )
+
+
+class LocationsEditView(BaseLoggedInView):
+    breadcrumb = BreadCrumb()
+    submit = Text("//form[contains(@id, 'edit')]//input[@name='commit']")
+    cancel = Text("//a[text()='Cancel']")
+
+    @property
+    def is_displayed(self):
+        breadcrumb_loaded = self.browser.wait_for_element(
+            self.breadcrumb, exception=False)
+        return (
+                breadcrumb_loaded
+                and self.breadcrumb.locations[0] == 'Locations'
+                and self.breadcrumb.read().starts_with('Edit ')
+        )
+
+    @View.nested
+    class primary(SatVerticalTab):
+        parent_id = FilteredDropdown(id='location_parent_id')
```

Comment:
why `parent_id` and not just `parent`?
<img width="510" alt="image" src="https://user-images.githubusercontent.com/11719030/42219379-6b043a6e-7ed4-11e8-9b22-c59fa4913d01.png">

Reply:
@abalakh With just parent it doesn't work, it looks like I rewrite parent somewhere else and then airgun browser want to fill parent but he doesn't have method fill and it will cause in widget not implemented error. Maybe you can explain to me which parent method I rewrote to better understanding.

Reply:
Oh, right. Every view or widget have `.parent` attr which basically points to either browser or another parent widget.
Maybe `parent_location` then? I'd never guess `parent` dropdown is called `parent_id` :)